main/src/ca/uwaterloo/flix/language/phase/Parser2.scala

```diff
@@ -174,16 +174,14 @@ object Parser2 {
           val child = stack.head
           val openToken = locationStack.head
           stack.head.loc = if (stack.head.children.length == 0)
-          // If the subtree has no children, give it a zero length position just after the last
-          // token.
+            // If the subtree has no children, give it a zero length position just after the last token.
```

Comment:
isn't most of the parser column width 100?

Reply:
Ah yes